wa/workloads/uibench/__init__.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+from wa import Parameter, ApkWorkload
+
+class Uibench(ApkWorkload):
+
+    name = 'uibench'
+    description = 'Runs a particular activity of the UIBench workload suite.'
+    package_names = ['com.android.test.uibench']
+    loading_time = 1
+
+    parameters = [
+        Parameter('activity', kind=str,
+            description='The UIBench activity to be run. Please not that the '
+            'default value of this parameter will result in UIBench opening '
+            'in its workload selection menu.'),
+        Parameter('duration', kind=int, default=10, description='As activities '
+            'do not finish, this workload will terminate UIBench after the '
+            'given duration.'),
+    ]
+
+    def setup(self, context):
+        if self.activity is not None:
+            if self.activity not in self.apk.apk_info.activities:
+                msg = 'Unknown activity "{}". Known activities:\n\t{}'
+                activities = '\n\t'.join(self.apk.apk_info.activities)
+                raise ValueError(msg.format(self.activity, activities))
+            self.apk.apk_info.activity = '.{}'.format(self.activity)
```

Comment:
```suggestion
            self.apk.activity = '.{}'.format(self.activity)
```
And, in `PackageHandler`:
```python
activity = property(lambda self: return None if self.apk_info is None else self.apk_info.activity,
                    lambda self, activity: self.apk_info.activity = activity)
```